src/Wizard.js

```diff
@@ -216,9 +215,9 @@ export default class Wizard extends Webform {
   render() {
     const ctx = this.renderContext;
 
-    if (this.component.key) {
-      ctx.panels.map(panel => {
-        if (panel.key === this.component.key) {
+    if (this.component.id) {
+      ctx.panels.forEach(panel => {
```

Comment:
`.forEach` is more idiomatic since we're applying side-effects and not returning a new array.